core/mono-samples/Android/AndroidSampleApp.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Runtime.InteropServices;
+
+// it's not part of the BCL but runtime needs it for native-to-managed callbacks in AOT
+// To be replaced with NativeCallableAttribute
+public class MonoPInvokeCallbackAttribute : Attribute
+{
+    public MonoPInvokeCallbackAttribute(Type delegateType) { }
+}
+
+public static class AndroidSampleApp
+{
+    [DllImport("native-lib")]
+    private extern static int myAge();
+
+    [DllImport("native-lib")]
+    private extern static void androidRegisterCounterIncrement(Func<string> del);
+    private static Func<string> incrementHandler = null;
```

Comment:
You can get rid of `incrementHandler` variable - we don't need to save static callback into it. (same for `nameHandler`)

Reply:
Thanks